tutorials/shaders/compute_shaders.rst

```diff
@@ -215,16 +313,19 @@ To execute our compute shader we just need to submit the pipeline to the GPU and
     rd.Submit();
     rd.Sync();
 
-Ideally, you would not synchronize the RenderingDevice right away as it will cause the CPU to wait for the GPU to finish working. In our example we synchronize right away because we want our data available for reading right away. In general, you will want to wait at least a few frames before synchronizing so that the GPU is able to run in parellel with the CPU.
-
-Congratulations you created and executed a compute shader. But wait, where are the results now?
+Ideally, you would not call ``sync()`` to synchronize the RenderingDevice right
+away as it will cause the CPU to wait for the GPU to finish working. In our
+example we synchronize right away because we want our data available for reading
+right away. In general, you will want to wait at least a few frames before
+synchronizing so that the GPU is able to run in parellel with the CPU.
```

Comment:
2-3 frames is a recommend delay for waiting:

```suggestion
Ideally, you would not call ``sync()`` to synchronize the RenderingDevice right
away as it will cause the CPU to wait for the GPU to finish working. In our
example, we synchronize right away because we want our data available for reading
right away. In general, you will want to wait at least 2-3 frames before
synchronizing so that the GPU is able to run in parellel with the CPU.
```